src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,35 @@
 package mate.academy;
 
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    private ConcurrentHashMap events = new ConcurrentHashMap();
```

Comment:
The `ConcurrentHashMap` is used without specifying generic types. This can lead to type safety issues. Consider specifying the key and value types, e.g., `ConcurrentHashMap<String, Event>`.